### PR TITLE
chore(iOS configs) configure iOS app PE-2065

### PR DESCRIPTION
**Pull request by thiagocarvalhodev**
- Update Podfile with the supported iOS version
- Apply settings upgrade from XCode (especially related to an older iOS version, update packages, and fix deprecates)
- Use the correct app name
- Use the correct bundle ID
- Add permissions
- Define device screen orientation

### CODE REVIEW

**Review by fedellen — APPROVED** [review]
LGTM

**Review by matibat — APPROVED** [review]
LGTM

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -26,11 +28,20 @@
 	<string>LaunchScreen</string>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
+		<string>remote-notification</string>
+	</array>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>ArDrive requires access to the photo library in order to facilitate photo and video uploads to your drives.</string>
```
Comment (arielmelendez) [inline ios/Runner/Info.plist:37]: Remind me: where do localizations for these eventually go?

**Review by arielmelendez — APPROVED** [review]
Looks great!

**Pull request merged**